### PR TITLE
feat : added wishlist

### PR DESCRIPTION
**Pull request by raman1999**
image

### CODE REVIEW

**Review by vivekmehto — APPROVED** [review]
Just a suggestion to adjust ``btn-dismiss`` apart from that it looks good, and good use of utility classes.

File: screens/wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./wishlist.css">
+    <title>Wishlist</title>
```
Comment (kjais1720) [inline screens/wishlist/wishlist.html]: Add your brand name in the title as well

File: screens/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="nav bg-radient">
+        <div class="nav-logo">
+            <span class="nav-header l-sp-1">Pure<span class="txt-theme txt-bold">Fitness</span></span>
```
Comment (kjais1720) [inline screens/wishlist/wishlist.html:14]: try to use descriptive classnames, here its not clear what does the `l-sp-1` class do

File: screens/wishlist/wishlist.html (hunk 1 above)
Comment (raman1999) [reply screens/wishlist/wishlist.html]: yes, I will do it

File: screens/wishlist/wishlist.html (hunk 2 above)
Comment (raman1999) [reply screens/wishlist/wishlist.html:14]: l-sp is for letter spacing

**Pull request merged**